Cakefile

```diff
@@ -45,15 +45,18 @@ task 'tests', "Run tests #{taskDetails}", (opts) ->
 
     env = if options['env'] then "NODE_ENV=#{options.env}" else "NODE_ENV=test"
     env += " USE_JS=true" if options['use-js']? and options['use-js']
+    env += " NAME=home TOKEN=token"
     logger.info "Running tests with #{env}..."
     command = "#{env} mocha " + files.join(" ") + " --reporter spec --colors "
     command += "--compilers coffee:coffee-script/register"
     exec command, (err, stdout, stderr) ->
         console.log stdout
-        if err or stderr
-            console.log stderr
-            console.log "Running mocha caught exception:\n" + err
-            setTimeout (-> process.exit 1), 100
+        if err?
+            console.log "Running mocha caught exception:\n#{err}"
+            process.exit 1
```

Comment:
you should setTimeout before process.exit, otherwise the console.log is lost sometimes